docs/consensus/consensus_chain.md

```diff
@@ -125,3 +125,25 @@ Justifications contain a set of all PoT checkpoints since the parent block up to
 3. Import and verify blocks. Ban bad peers.
 4. When you are close to tip (~18 blocks) switch to keep-up sync
 5. When get to tip start participate in consensus
+
+## Block Reward Address
+In the current blockchain farming setup, a farmer’s identity is used for plot creation, block signing, and receiving block rewards, posing risks like plot invalidation on testnet when used across multiple nodes and incompatibility with cold wallets. To address these, it's proposed to decouple the farmer's identity from the reward address by introducing a `--reward-address` argument in the farmer app, allowing the specification of a separate address for block rewards. This change, reflected in the updated `Solution` structure with an additional `reward_address` field, would enhance security, support multi-replica farming, and align farming operations with practices in PoW mining where reward addresses are independent of operational identities.
+
+```rust
+pub struct Solution<AccountId> {
+    /// Public key of the farmer that created the solution
+    pub public_key: AccountId,
++    /// Address for receiving block reward
++    pub reward_address: AccountId,
+    /// Index of encoded piece
+    pub piece_index: u64,
+    /// Encoding
+    pub encoding: Piece,
```

Comment:
Actually these `struct` fields are outdated. In the `proof_of_archival_storage.md` spec somewhere there should be the new one. Or double-check it with the codebase directly.